serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -0,0 +1,101 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-jwt"]
+= Using JSON Web Token authentication with {ProductShortName} and {ServerlessProductName}
+:context: serverless-ossm-jwt
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This section provides information about how you can enable JSON Web Token (JWT) authentication for Knative services.
+
+.Prerequisites
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+* Configure xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[{ProductShortName} with {ServerlessProductName}], including enabling sidecar injection for your Knative services.
+
+[IMPORTANT]
+====
+Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+====
+
+.Procedure
+
+. Create a policy in your serverless application namespace that only allows requests with valid JSON Web Tokens (JWT):
+
+.. Copy the following Policy resource into a YAML file:
++
+
+[source,yaml]
+----
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: default
+spec:
+  origins:
+  - jwt:
+      issuer: testing@secure.istio.io
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/jwks.json"
+      triggerRules:
+      - excludedPaths:
+        - prefix: /metrics
+        - prefix: /healthz
+  principalBinding: USE_ORIGIN
+----
+
++
+[IMPORTANT]
+====
+The paths `/metrics` and `/healthz` must be included in `excludedPaths` because they are accessed from system Pods in the `knative-serving` namespace.
+====
+
+.. Apply the Policy resource YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Verification steps
+
+. If you try to use a `curl` request to get the Knative service URL, it is denied.
++
+
+[source,terminal]
+----
+$ curl http://hello-example-default.apps.mycluster.example.com/
+----
+
++
+.Example output
++
+
+[source,terminal]
+----
+Origin authentication failed.
+----
+
+. Verify the request with a valid JWT.
+.. Get the valid JWT token by entering the following command:
++
+
+[source,terminal]
+----
+$ TOKEN=$(curl https://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/demo.jwt -s) && echo "$TOKEN" | cut -d '.' -f2 - | base64 --decode -
+----
+
+.. Access the service by using the valid token in the header.
```

Comment:
Not sure if you want to be consistent here with 2a and have a colon at the end of this step. Also, if you want to specify that the token is used in a `curl` request header:

> Access the service by using the valid token in the `curl` request header: